lib/note-info/index.tsx

```diff
@@ -188,9 +188,7 @@ function characterCount(content) {
 
 const { markdownNote, pinNote, toggleNoteInfo } = appState.actionCreators;
 
-const mapStateToProps = ({ appState: state, ui: { filteredNotes } }) => {
-  const noteIndex = Math.max(state.previousIndex, 0);
-  const note = state.note ? state.note : filteredNotes[noteIndex];
+const mapStateToProps = ({ ui: { note } }) => {
```

Comment:
excellent example of where our data can get out of sync, where we're using separate references to try and refer to the same data